sources/niUI/src/API/niUI/IMaterial.h

```diff
@@ -19,12 +19,12 @@ struct iShaderConstants;
  */
 
 //! Material flags.
-enum eMaterialFlags
-{
+enum eMaterialFlags {
```

Comment:
Please try to avoid committing "formatting only" changes, it makes reviewing harder. Eventually we'll add a proper .clang-format and mandate all the code to go through it.